src/scales/categoryScale.ts

```diff
@@ -50,6 +50,19 @@ export class Category extends Scale<string, number> implements TransformableScal
     this._outerPadding = Category._convertToPlottableOuterPadding(0.5, d3InnerPadding);
   }
 
+  /**
+   * Return a clone of this category scale without any included values providers.
```

Comment:
I'm glad you're encapsulating this into a method. Can you include in this comment some info about what **IS** included in the partial "clone" of this scale.